packages/nimble-components/src/table/specs/table-column-specs/table-column-anchor-hld.md

```diff
@@ -0,0 +1,121 @@
+# Anchor Table Column HLD
+
+## Problem Statement
+
+We need to support links in table cells. We will do this by creating a new column type.
+
+## Links To Relevant Work Items and Reference Material
+
+[GitHub issue: Table allows configuration of hyperlink column type (#1012)](https://github.com/ni/nimble/issues/1012)
+
+## Implementation / Design
+
+This is a fairly straightforward execution of our custom column type pattern. The anchor column will use two fields from the data records: one for the link's visible text (the "label") and one for the url (the "href"). The column configuration will contain the rest of the native anchor parameters: `hreflang`, `ping`, `referrerpolicy`, `rel`, `target`, `type`, and `download`. The assumption is that these should be the same for each link of a given column (if used at all). An alternative would be to have a `*FieldName` property for each of the native anchor parameters, but that seems unlikely to be useful.
+
+The column will also take an optional `placeholder` value to use when a record does not define a label.
+
+*   _Element name_: `nimble-table-column-anchor`
+*   _Attributes/properties_:
+    *   `labelFieldName`
+    *   `hrefFieldName`
+    *   `placeholder`
+    *   `hreflang`
+    *   `ping`
+    *   `referrerpolicy`
+    *   `rel`
+    *   `target`
+    *   `type`
+    *   `download`
+
+### Cell Template
+We will conditionally render either a `nimble-anchor` or a `span` of text.
+
+```html
+When cellRecord.label and cellRecord.html both present
+    <nimble-anchor
```

Comment:
All I have tried is modifying the table story (large data set) so that it has five link columns, then testing the scrolling. It seems to scroll just as well as without any link columns. Do you have any other suggestions for performance testing?

Reply:
That seems reasonable to me. @atmgrifter00, do you have any specific performance testing ideas related to column content? I believe you were the one to made the decision to go with `span` rather than `nimble-text-field` for the text column.

Reply:
No, it was more precautionary than anything else, trying to guard against possible performance penalties from the "wall of text" scenario (hundreds of cells on screen at once). It's very possible that with virtualization, this concern it effectively neutralized. Because it was so easy to use a `span` in place of a `nimble-text-field`, since we weren't going to leverage any of the features of the `nimble-text-field` (beyond the placeholder, which was easy enough to replicate), it felt like a reasonable decision at the time.